deadbelly/breathEZ
# Tm/feature/style header faq

### What does this PR do? 
>Clearly and concisely describe the feature. 

This PR adds styling to the `Header `& `FAQ ` components.

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

I had to move the assets folder in the src folder to have access to the images.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
`src/Components/Faq/Faq.js` lines: 2, 27-31;
`src/Components/Faq/_Faq.scss` lines: 1-24;
`src/Components/Header/_Header.scss` lines: 1-12;
`src/index.scss ` lines: 1-7;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch` the branch
2. run `npm i`
3. run `npm start`
4. Look at the header
5. Click on the FAQ

### Steps To Take
>Any questions and possible steps moving forward.
We are gonna need to polish it up but for now I think it looks good.

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
Closes #5 
Closes #3
